plugin/switch.vim

```diff
@@ -116,6 +116,12 @@ let g:switch_builtins =
       \     '(\([^()]\{-}\))\s*=>\s*{':                 'function(\1) {',
       \     '\(\k\+\)\s*=>\s*{':                        'function(\1) {',
       \   },
+      \   '
```

Comment:
There seems to be an extra quote here, which might be the thing breaking the tests :).